website/docusaurus.config.js

```diff
@@ -256,4 +256,12 @@ async function createConfig() {
   return config;
 }
 
-module.exports = createConfig;
+// module.exports = createConfig;
+
+module.exports = {
```

Comment:
You should add/modify fields of `module.exports` within the function definition of `createConfig`, which creates and manipulates `module.exports`. By commenting out the old line 259 where the function was called, you are missing out essential configs for building the website, as well as the remark plugins added for our custom code blocks.